pom.xml

```diff
@@ -16,7 +16,7 @@
 
     <groupId>org.gel.models</groupId>
     <artifactId>models</artifactId>
-    <version>${report.models.version}</version>
+    <version>${report.models.version}-cva-dev</version>
```

Comment:
@indikaraj I'm not sure we want this here.
@Antonior26, @priesgo, any thoughts ?